### PR TITLE
New content checks

### PR DESCRIPTION
**Pull request by jessica-zivtech**

### CODE REVIEW

File: tests/features/NewContentTypeSettings.feature [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@api
+Feature: New Content Type Settings
+  When I create a new content type
+  The preview before submitting being disabled
```
Comment (jodyHamilton) [inline tests/features/NewContentTypeSettings.feature]: weird sentence

File: tests/features/NewContentTypeSettings.feature [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  And the display author and date should be disabled
+  And the revisions should be enabled
+  And comments should be hidden
+  To make the process of creating new content types more efficient
```
Comment (jodyHamilton) [inline tests/features/NewContentTypeSettings.feature:8]: and to ensure new content types have the best default settings

**Comment by jessica-zivtech** [comment]
The checkbox in the failed test has a value of "revision" rather than "1" like the other checkboxes. Found this page which refers to a bug for checkboxes with values other than 0 and 1. https://github.com/minkphp/MinkGoutteDriver/issues/56

**Comment by jodyHamilton** [comment]
cool so it sounds like we need to update that jawn

File: tests/features/ContentCreation.feature [hunk 3]
```diff
@@ -7,18 +7,18 @@ Feature: Creating Basic Content
 Scenario: Administrator is able to log in and create Basic Page content
   Given I am logged in as a user with the "administrator" role
   When I visit "node/add/page"
-   And I fill in "Title" with "[random]"
+   And I fill in "Title" with "bottle"
```
Comment (jodyHamilton) [inline tests/features/ContentCreation.feature]: If we no longer are using random:
- remove use of random utility at top of FeatureContext.php
- remove fixStepArgument function from same file
- use more obvious placeholder name than 'bottle'

File: tests/features/bootstrap/FeatureContext.php [hunk 4]
```diff
@@ -109,261 +112,24 @@ public function transformLocalConf($argument) {
...
-  public function assertContentNotInWysiwygEditor($text, $tag, $region) {
-    $instance = $this->getWysiwygInstance($instanceId);
-    $content = $this->evaluateScript("return $instance.getContent()");
-    if (strpos($text, $content) !== FALSE) {
-      throw new \Exception(sprintf('The text "%s" was found in the "%s" WYSWIYG editor on the page %s', $text, $instanceId, $this->getSession()->getCurrentUrl()));
-    }
-  }
- 
-  /**
-  * @AfterScenario @deleteTestField
```
Comment (jodyHamilton) [inline tests/features/bootstrap/FeatureContext.php]: This shouldn't be deleted

**Pull request merged**